frontend/src/scenes/plugins/pluginActivityDescriptions.tsx

```diff
@@ -94,6 +95,37 @@ export function pluginActivityDescriber(logItem: ActivityLogItem): HumanizedChan
         }
     }
 
+    if (logItem.activity == 'export_success' && logItem.detail.trigger) {
+        const { dateFrom, dateTo } = logItem.detail.trigger.payload
+        const startDate = dayjs(dateFrom).format('YYYY-MM-DD')
+        // :TRICKY: Internally export date range is non-inclusive so transform it to be inclusive
+        const endDate = dayjs(dateTo).subtract(1, 'day').format('YYYY-MM-DD')
+
+        return {
+            description: (
+                <>
+                    Finished exporting historical events between {startDate} and {endDate} (inclusive).
+                </>
+            ),
+        }
+    }
+
+    if (logItem.activity == 'export_fail' && logItem.detail.trigger) {
+        const { dateFrom, dateTo } = logItem.detail.trigger.payload
+        const startDate = dayjs(dateFrom).format('YYYY-MM-DD')
+        // :TRICKY: Internally export date range is non-inclusive so transform it to be inclusive
+        const endDate = dayjs(dateTo).subtract(1, 'day').format('YYYY-MM-DD')
+
+        return {
+            description: (
+                <>
+                    Fatal error exporting historical events between {startDate} and {endDate} (inclusive). Check logs
+                    for more details.
+                </>
+            ),
+        }
+    }
```

Comment:
maybe sth like:
```suggestion
    if (logItem.activity.startsWith('export_') && logItem.detail.trigger) {
        const { dateFrom, dateTo } = logItem.detail.trigger.payload
        const startDate = dayjs(dateFrom).format('YYYY-MM-DD')
        // :TRICKY: Internally export date range is non-inclusive so transform it to be inclusive
        const endDate = dayjs(dateTo).subtract(1, 'day').format('YYYY-MM-DD')
        const description = (logItem.activity == 'export_success' ? 'Finished' : 'Fatal error') +  ' exporting historical events between {startDate} and {endDate} (inclusive).'

        return {
            description: (
                <>
                    {description}
                </>
            ),
        }
    }
```

Reply:
I don't see a reason why that's better - deduplicating markdown is not a huge win.

Reply:
mainly because you were doing something //TRICKY twice